components/AudioPlayer.js

```diff
@@ -0,0 +1,134 @@
+    setCurrentTime(progressBar.current.value);
+  }
+
+  const backThirty = () => {
+    progressBar.current.value = Number(progressBar.current.value - 30);
+    changeRange();
+  }
+
+  const forwardThirty = () => {
+    progressBar.current.value = Number(progressBar.current.value + 30);
+    changeRange();
+  }
+
+  return (
+    <section
+      className={`${utilsStyles.bg_white} ${styles.audio_player_section}`}
+      id="episodes"
+    >
+      <div className={styles.audio_player_section_container}>
+        <h2 className={utilsStyles.font_black}>
+          <span className={utilsStyles.font_accent}>E</span>pisodes
+        </h2>
+        <div className={styles.image_wrapper}>
+          <Image
+            priority
+            src="/assets/sample_profile_pic.jpeg"
+            layout="fill"
+            objectFit="cover"
+            objectPosition="left top"
+            alt="profile-dummy"
+            style={{boxShadow: "0 0 10 black"}}
+          />
+        </div>
+        {/* episode_info_section */}
+        <div className={styles.episode_info_section}>
+        <div className={styles.episode_title}>
+          <h3>003 Occupy</h3>
+          <h4>-Raid of Zuccati Park-</h4>
+          <h5>Nov.15th, 12-2am On Air</h5>
+        </div>
+        <div className={styles.episode_details}>
+          <p>Velit tincidunt ultricies dictum at. Amet, lectus gravida in enim proin mauris elit eu leo. Suspendisse pharetra donec erat aliquet consectetur lectus vitae, fermentum. Tempus, vulputate etiam in tristique volutpat vestibulum. Odio penatibus at vitae consectetur sed.</p>
+        </div>
+        </div>
+
+        <div className={styles.audioPlayer}>
+          <audio ref={audioPlayer} src="https://cdn.simplecast.com/audio/cae8b0eb-d9a9-480d-a652-0defcbe047f4/episodes/af52a99b-88c0-4638-b120-d46e142d06d3/audio/500344fb-2e2b-48af-be86-af6ac341a6da/default_tc.mp3" preload="metadata"></audio>
+          <div className={styles.audioPlayerPlaybutton}>
+          <button className={styles.forwardBackward} onClick={backThirty}><BsArrowLeftShort /> 30</button>
+          <button onClick={togglePlayPause} className={styles.playPause}>
+            {isPlaying ? <FaPause /> : <FaPlay className={styles.play} />}
+          </button>
+          <button className={styles.forwardBackward} onClick={forwardThirty}>30 <BsArrowRightShort /></button>
+          </div>
+
+          <div className={styles.audioPlayerBar}>
+          {/* current time */}
+          <div className={styles.currentTime}>{calculateTime(currentTime)}</div>
+          
+          {/* progress bar */}
+            <input type="range" className={styles.progressBar} defaultValue="0" ref={progressBar} onChange={changeRange} />
+          
+
+          {/* duration */}
+          <div className={styles.duration}>{(duration && !isNaN(duration)) && calculateTime(duration)}</div>
+
+          </div>
+        </div>
+      </div>
+    </section>
+  )
+}
+
+export { AudioPlayer }
```

Comment:
```suggestion
export default AudioPlayer
```
I assume that we export only `AudioPlayer`, so we can say `export default` like this.
@bluwakpa did you have any specific reason for it?